app/slackimport.go

```diff
@@ -218,10 +214,9 @@ func (a *App) SlackAddBotUser(teamId string, log *bytes.Buffer) *model.User {
 	if mUser := a.OldImportUser(team, &botUser); mUser != nil {
 		log.WriteString(utils.T("api.slackimport.slack_add_bot_user.email_pwd", map[string]interface{}{"Email": botUser.Email, "Password": password}))
 		return mUser
-	} else {
-		log.WriteString(utils.T("api.slackimport.slack_add_bot_user.unable_import", map[string]interface{}{"Username": username}))
-		return nil
 	}
+	log.WriteString(utils.T("api.slackimport.slack_add_bot_user.unable_import", map[string]interface{}{"Username": username}))
```

Comment:
nit: Can you reverse this conditional, to check `mUser == nil` first and having the successful case follow?